### ISSUE TITLE
compatibility with multiple runs

### ISSUE THREAD

**Issue by JWinawer** [vistalab/PRFmodel#55]
Currently, the vistasoft docker operates only one run (e.g., a bar scan, which might be the average of several acquisitions). However, one might have runs that cannot be averaged together, such as one set of runs with wedges and another with rings. The vistasoft Matlab code can handle this, but the docker cannot. The winawerlab would like to be able run dockers on multiple runs. How can we help implement this in the docker?

**Comment by garikoitz** [comment]
You mean that inside a subject's folder there will be, simplifying, two different nifti-s that will require two different stimuli files  and probably parameters? That will require changes on the analyze-base container that Noah wrote as well as changes on how we translate what comes from the container... I don't know if it won't be easier doing it outside using a symbolic link scheme... we can zoom and decide, what do you think?

**Comment by JWinawer** [comment]
Yes, one would input two (or more) different nifti files for the data and
the same number of nifti files for the stimulus. Not sure if different
parameters are needed, as one is still solving a single model per voxel,
and one can assume the same FOV for the scans. This is relatively common in
datasets. In the HCP dataset, there are bars, wedges, and ring scans, so
these three cannot be averaged together. I am not sure if what comes out of
the container would need to change - there is still one pRF solution per
voxel, no matter how many scans are input. Happy to zoom about it. Adding
@nben@uw.edu <nben@uw.edu> here. Let's move to private email to work out a
time.

On Wed, Jan 27, 2021 at 3:59 PM Garikoitz Lerma-Usabiaga <
notifications@github.com> wrote:

> You mean that inside a subject's folder there will be, simplifying, two
> different nifti-s that will require two different stimuli files and
> probably parameters? That will require changes on the analyze-base
> container that Noah wrote as well as changes on how we translate what comes
> from the container... I don't know if it won't be easier doing it outside
> using a symbolic link scheme... we can zoom and decide, what do you think?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vistalab_PRFmodel_issues_55-23issuecomment-2D768572658&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=W4u2U_DhoNoJOD0UdsYQiqH56Bg3zFfOtNa0RCuuEIs&m=0_zK4AFkzGcZ92iBR2HUknkXI4tVr-wknsjm_sL9N68&s=5S4ER68uXZY6iiucNSI1B0uqC7Lu8va90Aw0j5IrytM&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AAQFPXSPI3NTX5L5DGQYSFTS4B46RANCNFSM4WVRYCKA&d=DwMCaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=W4u2U_DhoNoJOD0UdsYQiqH56Bg3zFfOtNa0RCuuEIs&m=0_zK4AFkzGcZ92iBR2HUknkXI4tVr-wknsjm_sL9N68&s=QN044ktTKCHq_7NpAK5B1PXQSdijJo6Uqe2JNj2eXSQ&e=>
> .
>


-- 
Jonathan Winawer
Associate Professor of Psychology and Neural Science

New York University
6 Washington Place
New York, NY, 10003
(212) 998-7922 (phone)
(212) 995-4018 (fax)
jonathan.winawer@nyu.edu
http://psych.nyu.edu/winawer/
